core/src/transport/mod.rs

```diff
@@ -47,73 +46,85 @@ pub use self::choice::OrTransport;
 pub use self::memory::MemoryTransport;
 pub use self::upgrade::Upgrade;
 
-/// A transport is an object that can be used to produce connections by listening or dialing a
-/// peer.
+/// A transport is a connection-oriented communication channel between two peers,
```

Comment:
The transport itself is not a channel, but allows creating channels. I think this should be worded a bit differently.

Reply:
True, I will try to reword it.

Reply:
New suggestion in https://github.com/libp2p/rust-libp2p/pull/1005/commits/3216bae891144c5118fa90de9423ab3b878ceb6b.